src/Management/src/Endpoint/Actuators/CloudFoundry/CloudFoundrySecurityMiddleware.cs

```diff
@@ -101,13 +103,13 @@ public async Task InvokeAsync(HttpContext context)
 
     internal string GetAccessToken(HttpRequest request)
     {
-        if (request.Headers.TryGetValue(PermissionsProvider.AuthorizationHeaderName, out StringValues headerValue))
+        if (request.Headers.TryGetValue(HeaderNames.Authorization, out StringValues authorizationHeaderValue))
         {
-            string header = headerValue.ToString();
+            string authorizationValue = authorizationHeaderValue.ToString();
 
-            if (header.StartsWith(PermissionsProvider.BearerHeaderNamePrefix, StringComparison.OrdinalIgnoreCase))
+            if (authorizationValue.StartsWith(BearerTokenPrefix, StringComparison.OrdinalIgnoreCase))
             {
-                return header[PermissionsProvider.BearerHeaderNamePrefix.Length..];
+                return authorizationValue[BearerTokenPrefix.Length..];
```

Comment:
In method `CloudFoundrySecurityMiddleware.ReturnErrorAsync`:

![image](https://github.com/user-attachments/assets/0cdcb36a-c8d3-4ecf-9d51-b8dd96d8433a)

Is this a bug? The comment seems to contradict the code.

Reply:
The code says "If we should `UseStatusCodeFromResponse` or the error code is less than 500, we should use the status code from the response". The comment is saying "Hi friend, that's nice that you think you don't want to see a 403, but we're going to show it to you anyway because you'd be super confused if we return 200" -- `UseStatusCodeFromResponse` is for IIS/HWC compatibility reasons (and the built-in server error pages that break our ability to return a response body)

Reply:
Can you clarify the comment in code, and link to the related issue?